runtime/common/BaseRestRemoteClient.h

```diff
@@ -276,6 +287,12 @@ class BaseRemoteRestRuntimeClient : public cudaq::RemoteRuntimeClient {
               const std::string &backendSimName, const std::string &kernelName,
               void (*kernelFunc)(void *), void *kernelArgs,
               std::uint64_t argsSize, std::string *optionalErrorMsg) override {
+    if (isDisallowed(io_context.name)) {
+      throw std::runtime_error(
+          io_context.name +
+          " operation is not supported with cudaq target remote-mqpu!");
+    }
+
```

Comment:
Nit: per the [LLVM coding standards](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements) that we mostly follow, the braces for this `if` statement are not required and should be removed.